code/modules/research/xenoarchaeology/artifact/effects/unknown_effect_projectiles.dm

```diff
@@ -0,0 +1,45 @@
+/var/global/list/validartifactprojectiles = list(
+	/obj/item/projectile/beam,
+	/obj/item/projectile/beam/retro,
+	/obj/item/projectile/beam/practice,
+	/obj/item/projectile/beam/practice/stormtrooper,
+	/obj/item/projectile/beam/lightlaser,
+	/obj/item/projectile/beam/heavylaser,
+	/obj/item/projectile/beam/xray,
+	/obj/item/projectile/beam/pulse,
+	/obj/item/projectile/beam/lasertag/omni,
+	/obj/item/projectile/beam/bison,
+	/obj/item/projectile/energy/electrode,
+	/obj/item/projectile/energy/declone,
+	/obj/item/projectile/energy/bolt/large,
+	/obj/item/projectile/energy/plasma/pistol,
+	/obj/item/projectile/energy/plasma/MP40k,
+	/obj/item/projectile/energy/rad,
+	/obj/item/projectile/energy/buster,
+	/obj/item/projectile/ion,
+	/obj/item/projectile/temp,
+	/obj/item/projectile/kinetic,
+	/obj/item/projectile/forcebolt
+)
```

Comment:
I tried, lightning seems to be linked to the spell too much to easily use as a normal projectile